### PR TITLE
Add names of LHCb software packages + TISTOS

### PR DESCRIPTION
**Pull request by abmorris**

### CODE REVIEW

File: glossary/b.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # B
 
+## Bender
+
```
Comment (eduardo-rodrigues) [inline glossary/b.md]: For here and other places: please end with a dot.

File: glossary/e.md [hunk 2]
```diff
@@ -22,6 +22,10 @@ This includes all devices in the areas of data acquisition, detector control (e.
 and the interaction with the outside world.
 Details in the Online System [webpages](http://lhcb-online.web.cern.ch/lhcb-online/ecs/default.htm).
 
+## Erasmus
+
+High-level physics analysis software repository, similar to [Urania](e.md#Urania) but based on [Bender](b.md#Bender)/[DaVinci](d.md#DaVinci).
```
Comment (eduardo-rodrigues) [inline glossary/e.md]:
```suggestion
High-level physics analysis software repository, similar to [Urania](u.md#Urania) but based on [Bender](b.md#Bender)/[DaVinci](d.md#DaVinci).
```

File: glossary/m.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # M
 
+## Moore
+
+The LHCb trigger emulation application.
```
Comment (eduardo-rodrigues) [inline glossary/m.md]:
```suggestion
The LHCb trigger emulation application. [Project website](http://lhcbdoc.web.cern.ch/lhcbdoc/moore/) .
```

File: glossary/g.md [hunk 4]
```diff
@@ -8,6 +8,10 @@
 
 The **LHCb** software framework, see the [Gaudi homepage](http://gaudi.web.cern.ch/gaudi/) and the [**Starterkit** high level overview](https://lhcb.github.io/starterkit-lessons/first-analysis-steps/davinci.html) for more details.
 
+## Gauss
+
+The LHCb simulation application. [Project website](http://cern.ch/LHCb-release-area/DOC/gauss/)
```
Comment (eduardo-rodrigues) [inline glossary/g.md]:
```suggestion
The LHCb simulation application. [Project website](http://lhcbdoc.web.cern.ch/lhcbdoc/gauss/).
```

File: glossary/d.md [hunk 5]
```diff
@@ -1,5 +1,9 @@
 # D
 
+## DaVinci
+
+Event selection and data analysis application. [Project website](http://cern.ch/LHCb-release-area/DOC/davinci/)
```
Comment (eduardo-rodrigues) [inline glossary/d.md]:
```suggestion
Event selection and data analysis application. [Project website](lhcbdoc.web.cern.ch/lhcbdoc/davinci/).
```

File: glossary/b.md [hunk 6]
```diff
@@ -1,5 +1,17 @@
...
+Python-based physics analysis environment. [Project website](http://lhcbdoc.web.cern.ch/lhcbdoc/bender/)
+
 ## Bookkeeping
 
-The LHCb data recording and data provenance tool. Every single file ever produced by LHCb centrally-managed productions (p.md#Productions) is recorded by the LHCb Bookkeeping. Users can browse the contents of the Bookkeeping via the [LHCbDIRAC Web Portal](https://lhcb-portal-dirac.cern.ch/DIRAC/?view=tabs&theme=Grey&url_state=1|*LHCbDIRAC.BookkeepingBrowser.classes.BookkeepingBrowser).
+The LHCb data recording and data provenance tool. Every single file ever produced by LHCb centrally-managed [productions](p.md#Productions) is recorded by the LHCb Bookkeeping. Users can browse the contents of the Bookkeeping via the [LHCbDIRAC Web Portal](https://lhcb-portal-dirac.cern.ch/DIRAC/?view=tabs&theme=Grey&url_state=1|*LHCbDIRAC.BookkeepingBrowser.classes.BookkeepingBrowser).
+
+## Boole
+
+The LHCb digitisation (simulation of detector response) application. [Project website](http://cern.ch/LHCb-release-area/DOC/boole/)
```
Comment (eduardo-rodrigues) [inline glossary/b.md]:
```suggestion
The LHCb digitisation (simulation of detector response) application. [Project website](lhcbdoc.web.cern.ch/lhcbdoc/boole/).
```

File: glossary/b.md [hunk 7]
```diff
@@ -1,5 +1,17 @@
...
+
+## Brunel
+
+The LHCb reconstruction application. [Project website](http://cern.ch/LHCb-release-area/DOC/brunel/)
```
Comment (eduardo-rodrigues) [inline glossary/b.md]:
```suggestion
The LHCb reconstruction application. [Project website](lhcbdoc.web.cern.ch/lhcbdoc/brunel/).
```

File: glossary/p.md [hunk 8]
```diff
@@ -1,5 +1,9 @@
 # P
 
+## Panoramix
+
+Graphical display of detector and event infomration. [Project website](http://lhcbdoc.web.cern.ch/lhcbdoc/panoramix/)
```
Comment (eduardo-rodrigues) [inline glossary/p.md]:
```suggestion
Graphical display of detector and event information. [Project website](http://lhcbdoc.web.cern.ch/lhcbdoc/panoramix/).
```

File: glossary/t.md [hunk 9]
```diff
@@ -13,14 +13,24 @@ See [Flavour Tagging](f.md#flavourtagging).
...
 
 A unique 32-bit configuration value (in base 16) representing a given trigger configuration (sequence of algorithms and selection cuts).
 
 ## Tier (Grid) {#Tier}
 
 Grid sites are split into tiers depending on their resources. Tier 1 (i.e. RAL, PIC) and tier 2 sites (mostly universities) store data, the tier 1s in particular storing raw data. The tier 3 sites do not store data. See [here for details](http://wlcg-public.web.cern.ch/tier-centres).
 
+## TISTOS
+
+Trigger decisions can be classified into three categories
```
Comment (eduardo-rodrigues) [inline glossary/t.md]:
```suggestion
Trigger decisions can be classified into three categories:
```

File: glossary/t.md [hunk 10]
```diff
@@ -13,14 +13,24 @@ See [Flavour Tagging](f.md#flavourtagging).
...
+
+- Trigger on signal (TOS): a positive decision can be reached using only information from the signal candidate.
```
Comment (eduardo-rodrigues) [inline glossary/t.md]:
```suggestion
- Trigger On Signal (TOS): a positive decision can be reached using information from the signal candidate.
```

Comment (eduardo-rodrigues) [reply glossary/t.md]: The fact that one has the TOB is saying that TOS does not imply TOB. So the word "only" is incorrect.

File: glossary/t.md [hunk 11]
```diff
@@ -13,14 +13,24 @@ See [Flavour Tagging](f.md#flavourtagging).
...
+- Trigger independently of signal (TIS): a positive decision can be reached using only information from the rest of the event.
```
Comment (eduardo-rodrigues) [inline glossary/t.md]:
```suggestion
- Trigger Independently of Signal (TIS): a positive decision can be reached using information from the rest of the event.
```

File: glossary/t.md [hunk 12]
```diff
@@ -13,14 +13,24 @@ See [Flavour Tagging](f.md#flavourtagging).
...
+- Trigger on both (TOB): a positive decision requires the entire event.
+
+TIS and TOS are not mutually exclusive.
```
Comment (eduardo-rodrigues) [inline glossary/t.md]:
```suggestion
Note that the TIS and TOS categories are not mutually exclusive.
```

File: glossary/u.md [hunk 13]
```diff
@@ -5,6 +5,10 @@
 Unknown small object (think of a grain of dust) that is falling within the LHC beam pipe into the beam, leading to large interactions between the beam and the object, and thus particle showers.
 UFOs are known to cause LHC beam dumps upon detection, as a protective measure.
 
+## Urania
+
+High-level physics analysis software repository, similar to [Erasmus](e.md#Erasmus) but based on heptools.
```
Comment (eduardo-rodrigues) [inline glossary/u.md]: What is heptools?

Comment (abmorris) [reply glossary/u.md]: I don't know. I got it from the description on GitLab: https://gitlab.cern.ch/lhcb/Urania

Comment (eduardo-rodrigues) [reply glossary/u.md]: Then better to remove that bit, since uncertain.

**Pull request merged**